sympy/matrices/dense.py

```diff
@@ -1281,16 +1281,16 @@ def hessian(f, varlist, constraints=[]):
     return out
 
 
-def jordan_cell(eigenval, n):
+def jordan_block(eigenval, n):
```

Comment:
I mean that `from matrices.dense import jordan_cell` must still work, but it should raise a deprecation warning.